docs/source/user_guide/dpf_concepts.rst

```diff
@@ -36,3 +37,11 @@ DPF concepts
        :text-align: center
 
        .. image:: ../images/drawings/checklist.png
+
+    .. card:: Using DPF across a network
```

Comment:
I'm not sure about the title naming "across a network" because you are also presenting the in process server no?

Reply:
@cbellot000 initially I wanted to indicate when people would need to know about this concept, which would be when they do not want the default InProcess, which would be when wanting to do remote operations. But I guess I can find something more generic.

Reply:
```suggestion
    .. card:: Using DPF locally or across a network
```

Reply:
```suggestion
    .. card:: Using DPF: locally/remotely
```